UQ-eLIPSE/crucible-components
# Story/VETHUB-22

Changing button output to finish when it is the final question.

Button has been added to its own file as a component which emits the submit function.

Since the data currently contains 1 question in this context, it will change the text to finished once the user answers. This functionality will be expanded as we focus on the quiz component which will include the state of next(and skip).

Some changes have been made to accommodate a logic for the finished state which includes adding the current object in an array to be loopable.

To test:

- Run `yarn dev` and then navigate to https://localhost:5173
- Click the option.
- Click submit
- Verify you see the finished button.
- Run `yarn test` to run the updated tests.
